app/models/manageiq/providers/cloud_manager/vm_or_template.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::CloudManager::VmOrTemplate < ActsAsArScope
+  delegate :all_orphans, :all_archived, :to => :aar_scope
+  delegate :klass, :to => :aar_scope, :prefix => true
+
+  def ar_scope
```

Comment:
Also, it's self.aar_scope for infra and the delegates are at the class level, but different here.